src/creature.h

```diff
@@ -186,6 +205,15 @@ class Creature
                                                    double total_dispersion );
 
         /**
+         * Makes an aiming/attack roll for a single projectile attack shot.
+         *
```

Comment:
Style: no line break between preamble and following lines. Also no stop at end of preamble sentence